cmd/events.go

```diff
@@ -63,6 +66,9 @@ func init() {
 	triggerCmd.Flags().BoolVarP(&isAnonymous, "anonymous", "a", false, "Denotes if the event is anonymous. Only applies to Gift and Sub events.")
 	triggerCmd.Flags().IntVarP(&count, "count", "c", 1, "Count of events to trigger. This will simulate a sub gift, or large number of cheers.")
 	triggerCmd.Flags().StringVarP(&secret, "secret", "s", "", "Webhook secret. If defined, signs all forwarded events with the SHA256 HMAC.")
+	triggerCmd.Flags().StringVarP(&status, "status", "S", "", "Status of the event object, currently applies to channel points redemptions.")
+	triggerCmd.Flags().StringVarP(&objectId, "object-id", "i", "", "Manually set the ID of the event payload object (for example the reward ID in redemption events).")
```

Comment:
Do you mean the ID of the channel points object?

Reply:
If so- I believe that item-id might be a bit better for outside users- object-id is a bit obtuse in my opinion.

Reply:
Yeah, I wanted to name it as generic as possible, so the parameter can be reused for other calls, in the future.
But "item-id" indeed sounds like a better name.